cloudify_cli/commands/agents.py

```diff
@@ -225,3 +242,74 @@ def worker(dep_id):
         ))
 
         raise SuppressedCloudifyCliError()
+
+
+@agents.command(name='transfer',
+                short_help='Configure agents to work with a new'
+                           ' Cloudify-Manager [Manager only]')
+@cfy.argument('deployment-id', required=False)
+@cfy.options.include_logs
+@cfy.options.verbose()
+@cfy.options.tenant_name_for_list(
+    required=False, resource_name_for_help='relevant deployment(s)')
+@cfy.options.all_tenants
+@cfy.options.manager_ip
+@cfy.options.manager_certificate
+@cfy.options.manager_rest_token
+@cfy.pass_logger
+@cfy.pass_client()
+def transfer(deployment_id,
+             include_logs,
+             tenant_name,
+             logger,
+             client,
+             all_tenants,
+             manager_ip,
+             manager_certificate,
+             manager_rest_token):
+    """Configure agents to work with a new Cloudify-Manager.
+
+    `DEPLOYMENT_ID` - The ID of the deployment you would like to
+    configure agents in.
+
+    """
+    with open(manager_certificate, 'r') as ssl_file:
```

Comment:
Need to add verification here that the path exists before opening, and raise a helpful error